### DISCUSSION TITLE
Issues?

### DISCUSSION THREAD

**Discussion by chrisjj** [community/community#125789] [Codespaces]
### Select Topic Area

Question

### Body

Where's the place for reading, posting and discussing Codespace issues?

**Comment by pedrolemos4** [comment]
Under 'Github Product Categories'

**Comment by ebndev** [comment]
Hi @chrisjj, thanks for being a part of the GitHub Community!

I've gone ahead and moved this post into our [Codespaces](https://github.com/orgs/community/discussions/categories/codespaces) category for you. Please feel free to start another discussion there with your questions / comments.

**Comment by chrisjj** [comment]
> I've gone ahead and moved this post into our [Codespaces](https://github.com/orgs/community/discussions/categories/codespaces) category for you.

Strange. Because this page says I asked this question in Codespaces.

> Please feel free to start another discussion there with your questions / comments.

Just the one question. Still. Where's the place for reading, posting and discussing Codespace issues? That's issues. Thanks.

**Reply by ebndev** [reply]
I had moved this post from the General category to the Codespaces category. If you click on the link above, it will take you to the category where you can see discussions about Codespaces. Is that what you were looking for? Let me know if not!

**Reply by chrisjj** [reply]
No. Specifically issues. E.g. https://github.com/issues
